src/AppInstallerCLIE2ETests/InstallCommand.cs

```diff
@@ -8,118 +8,153 @@ namespace AppInstallerCLIE2ETests
 
     public class InstallCommand
     {
-        // Todo: this should point to a loopback address. Disabling the install tests until we have loopback support done in our e2e tests.
         // Todo: add unicode test cases after install tests are enabled.
-        private const string InstallTestSourceUrl = @"https://github.com/microsoft/appinstaller-cli/raw/master/src/AppInstallerCLIE2ETests/TestData";
+        private const string InstallTestSourceUrl = @"https://localhost:5001/TestKit";
         private const string InstallTestSourceName = @"InstallTestSource";
 
         private const string InstallTestExeInstalledFile = @"TestExeInstalled.txt";
-        private const string InstallTestMsiInstalledFile = @"AppInstallerTestExeInstaller.exe";
+        private const string InstallTestMsiInstalledFile = @"AppInstallerTestMsiInstaller.msi";
         private const string InstallTestMsiProductId = @"{A5D36CF1-1993-4F63-BFB4-3ACD910D36A1}";
         private const string InstallTestMsixName = @"6c6338fe-41b7-46ca-8ba6-b5ad5312bb0e";
 
-        //[SetUp]
+        [SetUp]
         public void Setup()
         {
             Assert.AreEqual(Constants.ErrorCode.S_OK, TestCommon.RunAICLICommand("source add", $"{InstallTestSourceName} {InstallTestSourceUrl}").ExitCode);

```

Comment:
>Assert.AreEqual(Constants.ErrorCode.S_OK, TestCommon.RunAICLICommand("source add", $"{InstallTestSourceName} {InstallTestSourceUrl}").ExitCode); [](start = 12, length = 144)

You need to remove the default winget source as well to ensure that no package comes along through it and creates a conflict.

Reply:
Removed Default winget source at test setup for install command and re-Added default winget source during teardown.